examples/cpp/Makefile

```diff
@@ -26,13 +32,17 @@ CXXFLAGS += -DEIGEN
 INCPATH += -I$(EIGEN)
 else
 ifdef SUPERLU
-LIBS += -L$(SUPERLU)/lib -Wl,-rpath,$(SUPERLU)/lib -lsuperlu
+LIBS += -L$(SUPERLU)/lib -Wl,-rpath,$(SUPERLU)/lib -lsuperlu #for non mac users.
+#LIBS += -L$(SUPERLU)/lib -Wl,-rpath,$(SUPERLU)/lib -lsuperlu -L/opt/homebrew/Cellar/superlu/7.0.0/lib #for mac os
 else
-LIBS += -lsuperlu
+LIBS += -lsuperlu #for non mac users.
+#LIBS += -lsuperlu -L/opt/homebrew/Cellar/superlu/7.0.0/lib #for mac os
 endif
 endif
 
-all: transport1D schrodinger1D elliptic1D elliptic2D convection_diffusion RK2
+
+all: transport1D schrodinger1D elliptic1D elliptic2D parabolic1D convection_diffusion RK2
```

Comment:
Everything from here down looks good. :)

Reply:
Hello @jbrzensk is there any other changes i need to make?